clients/web/src/vite-env.d.ts

```diff
@@ -0,0 +1,5 @@
+/// <reference types="vite/client" />
```

Comment:
let's remove unused codes

Reply:
updated in a3d06124eeaeb1aff9ed3fd02c4eadf7f7467947